org.aegisub.Aegisub.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.aegisub.Aegisub
+runtime: org.gnome.Platform
+runtime-version: '3.30'
```

Comment:
Update runtime 3.32

Reply:
GNOME 3.30 was the latest release when I submitted this application (December 2018, over half a year ago!).
I don't see the point in updating it if it's not going to be merged anytime soon.
I can upgrade GNOME to 3.32, but that's it. Don't count on further updates if this PR will still be blocked.